### PR TITLE
homepage's chinese content

### PR DESCRIPTION
**Pull request by KNGJOEL**

### CODE REVIEW

**Comment by KNGJOEL** [comment]
Screen Shot 2021-08-10 at 3 00 57 PM
Screen Shot 2021-08-10 at 3 01 23 PM
Screen Shot 2021-08-10 at 3 01 52 PM
Screen Shot 2021-08-10 at 3 02 04 PM
Screen Shot 2021-08-10 at 3 02 16 PM
Screen Shot 2021-08-10 at 3 02 28 PM
Screen Shot 2021-08-10 at 3 02 52 PM

**Comment by rebeccabilbro** [comment]
Thanks for working on making our home page available in Chinese @KNGJOEL! This seems like a strong start. 

@tl6kk &mdash; would you mind taking a look to see if everything seems ok? 

The main thing I'm noticing is that I think we prefer to use `zh` to `cn`.

To build the branch locally:

```bash
git clone git@github.com:rotationalio/rotational.io.git
cd rotationalio.github.io
git remote add KNGJOEL git@github.KNGJOEL/rotational.io.git
git fetch KNGJOEL
git checkout -b zh-home KNGJOEL/add-chinese-language-content-for-homepage
hugo serve -D
```

**Comment by tl6kk** [comment]
Thanks for your work @KNGJOEL for our Chinese viewers! The page looks great. I only noticed a few small errors:

- Under "Consulting Services" ("咨询服务“), the last row, there is an extra I before 数据服务
- The GDS name are in Franch
- This is really minor, but under "Software Development" ("软件开发"), the last row misses some space. Ideally should be "云 & 分布式系统“
- Our Team ”我们的团队“, there is an extra N in the beginning.

Everything else looks great to me!

**Comment by KNGJOEL** [comment]
Got it i'll solve it

**Review by rebeccabilbro — CHANGES_REQUESTED** [review]
Thank you @tl6kk for catching those edits and @KNGJOEL for making those updates so quickly! There are a few other small requested changes I've noted in my review, but with these additional edits I think we should be able to merge soon and get this page live!

@KNGJOEL am I correct in assuming that we will need to merge #37 before we merge #38?

File: data/zh/feature.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+      # feature item loop
+    - title : Rapide et Flexible
+      icon : ti-bolt # themify icon pack : https://themify.me/themify-icons
+      content : Le consensus hiérarchique est rapide et flexible, ce qui est essentiel pour les applications de nouvelle génération construites avec 5G, AI/ML, IoT, AR/VR et tout ce qui va suivre!
+
+      # feature item loop
+    - title : Facile à Utiliser
+      icon : ti-layout-grid2 # themify icon pack : https://themify.me/themify-icons
+      content : Nous donnons aux développeurs un accès à une cohérence forte à un niveau supérieur de la pile d'applications, ajoutant flexibilité et visibilité à vos services de données gérés existants.
```
[the remark this answers is not in the document]
Comment (KNGJOEL) [reply data/zh/feature.yml:27]: Understood, I will

**Comment by KNGJOEL** [comment]
> Thank you @tl6kk for catching those edits and @KNGJOEL for making those updates so quickly! There are a few other small requested changes I've noted in my review, but with these additional edits I think we should be able to merge soon and get this page live!
> 
> @KNGJOEL am I correct in assuming that we will need to merge #37 before we merge #38?

yes we will

**Review by rebeccabilbro — APPROVED** [review]

**Pull request merged**